lib/widgets/atoms/profile_header.dart

```diff
@@ -18,53 +19,68 @@ class ProfileHeader extends StatelessWidget {
+          Insets.paddingExtraLarge, Insets.paddingExtraLarge, 0),
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.center,
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
-          Flexible(
-            child: Padding(
-              padding: const EdgeInsets.all(Insets.paddingExtraSmall),
-              child: Text(
-                profileMessage,
-                textAlign: TextAlign.start,
-                softWrap: true,
-                style: theme.textTheme.titleLarge?.copyWith(
-                  color: theme.colorScheme.onSurface,
-                ),
-              ),
-            ),
-          ),
           Stack(
             children: [
-              if (profileCompletionPercentage != null)
-                SizedBox(
-                  width: Radii.avatarRadiusSmall * 2,
-                  height: Radii.avatarRadiusSmall * 2,
-                  child: CircularProgressIndicator(
-                    value: profileCompletionPercentage! / 100,
+              Positioned(
+                top: Insets.paddingExtraSmall,
+                child: ClipRRect(
+                  borderRadius:
+                      BorderRadius.circular(Radii.roundedRectRadiusSmall),
+                  child: Image(
+                    image: backgroundImage,
+                    width: Insets.paddingExtraLarge,
+                    height: Insets.paddingExtraLarge,
+                    fit: BoxFit.cover,
                   ),
                 ),
-              InkWell(
-                onTap: () {
-                  //TODO(m-rosario): Remove this once the real logout flow is defined by UX
-                  Provider.of<ScaffoldModel>(
-                    context,
-                    listen: false,
-                  ).setTestHomeScaffold();
-                  Scaffold.of(context).openDrawer();
-                },
-                child: CircleAvatar(
-                  radius: Radii.avatarRadiusSmall,
-                  backgroundImage: avatarUrl != null
-                      ? NetworkImage(avatarUrl!) as ImageProvider<Object>
-                      : const AssetImage(Assets.blankAvatar),
+              ),
+              Flexible(
+                child: Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                      Insets.paddingSmall + Insets.paddingExtraLarge,
+                      Insets.paddingSmall,
+                      Insets.paddingSmall,
+                      Insets.paddingSmall),
+                  child: Text(
+                    profileMessage,
+                    textAlign: TextAlign.start,
+                    softWrap: true,
+                    style: theme.textTheme.titleLarge?.copyWith(
+                      color: theme.colorScheme.onSurface,
+                      fontWeight: FontWeight.w400,
+                    ),
+                  ),
                 ),
               ),
             ],
           ),
```

Comment:
Flex only works for child widgets of a Column or Row. Putting this Flexible inside the Stack negates the flex, which again causes it to overflow:

<img width="343" alt="Screenshot 2023-10-03 at 10 26 13 AM" src="https://github.com/micromentor-team/mm-flutter-app/assets/25093428/f5b0186c-3bb5-4d52-879e-41af0553dd83">

I cannot think of any reason these widgets needs to be inside a stack, so maybe we can just rid of it.

Reply:
Hi Miguel, 
The reason I chose to introduce the "Stack" was because I wanted use mainAxisAignment.spaceBetween property to create space between the "Stack" of Positioned and Flexible widgets as well as the three dot menu on the right. Is there a better way to make sure the three dot menu is on the right in a row? I'll also continue to investigate and would appreciate insights.

Reply:
This can be accomplished easily with a Row I believe. Just place the Row children in the correct order from left to right and make sure that the child in the middle (the text) has a flex value so that it takes all of the available space, pushing the three dot menu to the right. Row can also use the spaceBetween property, but it won't be necessary if you use flex to get rid of space between widgets.

Reply:
Thanks Miguel. Appreciate the insight. Will play with the flex value and report back